This fix adds hiding of default datetime field in attribute swatches configurations if attribute is of other type.

diff --git a/src/module-elasticsuite-swatches/view/adminhtml/web/js/product-attributes.js b/src/module-elasticsuite-swatches/view/adminhtml/web/js/product-attributes.js
index bff2d73..0a9d3d6 100644
--- a/src/module-elasticsuite-swatches/view/adminhtml/web/js/product-attributes.js
+++ b/src/module-elasticsuite-swatches/view/adminhtml/web/js/product-attributes.js
@@ -40,6 +40,7 @@ define([
                 defaultValueText: $('#default_value_text'),
                 defaultValueTextarea: $('#default_value_textarea'),
                 defaultValueDate: $('#default_value_date'),
+                defaultValueDatetime: $('#default_value_datetime'),
                 defaultValueYesno: $('#default_value_yesno'),
                 isGlobal: $('#is_global'),
                 useProductImageForSwatch: $('#use_product_image_for_swatch'),
@@ -188,6 +189,7 @@ define([
                         defaultValueTextVisibility = false,
                         defaultValueTextareaVisibility = false,
                         defaultValueDateVisibility = false,
+                        defaultValueDatetimeVisibility = false,
                         defaultValueYesnoVisibility = false,
                         scopeVisibility = true,
                         useProductImageForSwatch = false,
@@ -213,6 +215,10 @@ define([
                             defaultValueDateVisibility = true;
                             break;
 
+                        case 'datetime':
+                            defaultValueDatetimeVisibility = true;
+                            break;
+
                         case 'boolean':
                             defaultValueYesnoVisibility = true;
                             break;
@@ -266,6 +272,7 @@ define([
                                     defaultValueTextVisibility = false;
                                     defaultValueTextareaVisibility = false;
                                     defaultValueDateVisibility = false;
+                                    defaultValueDatetimeVisibility = false;
                                     defaultValueYesnoVisibility = false;
                                     break;
 
@@ -289,6 +296,7 @@ define([
                     this.setRowVisibility(this.defaultValueText, defaultValueTextVisibility);
                     this.setRowVisibility(this.defaultValueTextarea, defaultValueTextareaVisibility);
                     this.setRowVisibility(this.defaultValueDate, defaultValueDateVisibility);
+                    this.setRowVisibility(this.defaultValueDatetime, defaultValueDatetimeVisibility);
                     this.setRowVisibility(this.defaultValueYesno, defaultValueYesnoVisibility);
                     this.setRowVisibility(this.isGlobal, scopeVisibility);
 
